versioned_docs/version-v1.3/troubleshooting/host.md

```diff
@@ -0,0 +1,139 @@
+---
+sidebar_position: 6
+sidebar_label: Host
+title: "Host"
+---
+
+<head>
+  <link rel="canonical" href="https://docs.harvesterhci.io/v1.3/troubleshooting/host"/>
+</head>
+
+## Node in Maintenance Mode Becomes Stuck in Cordoned State
+
+When you enable Maintenance Mode on a node using the Harvester UI, the node becomes stuck in the `Cordoned` state and the menu shows the **Enable Maintenance Mode** option instead of **Disable Maintenance Mode**.
```

Comment:
```suggestion
When you enable `Maintenance Mode` on a node using the Harvester UI, the node becomes stuck in the `Cordoned` state and the menu shows the **Enable Maintenance Mode** option instead of **Disable Maintenance Mode**.
```